### PR TITLE
Change logging order in compile-vue-templates.

### PR DESCRIPTION
**Pull request by kberg**
I wanted to give a good example here, but for some reason I don't know enough to force it to happen.

(I also renamed the primary method.)

### CODE REVIEW

File: compile-vue-templates.js [hunk 1]
```diff
@@ -175,123 +174,125 @@ checkComponent(
...
-checkComponent(
+generateVueClass(
   'src/components/help/HelpStandardProjects',
   require('./build/src/components/help/HelpStandardProjects').HelpStandardProjects,
   [],
 );
 
-function checkComponent(name, component, dataProperties) {
+function generateVueClass(name, component, dataProperties) {
+  process.stdout.write(`generating the Vue typed template file for ${name} ... `);
```
Comment (alrusdi) [inline compile-vue-templates.js:294]: Do we really need this output? It is kinda annoying to me :-)

Comment (kberg) [reply compile-vue-templates.js:294]: I hear you. I wouldn't mind removing it, but without anny logging, when it fails I wouldn't know which class failed to generate.

And before this PR, you wouldn't know which class it was generating _when the error occurred._ The user would only know that the file _before it_ compiled successfully.

How annoying is it? If it's that annoying, I will follow up with a PR that keeps it silent UNLESS there are errors. How about that? Deal?

Comment (alrusdi) [reply compile-vue-templates.js:294]: Yeah! It would be great!

Comment (kberg) [reply compile-vue-templates.js:294]: Will do.

**Review by alrusdi — APPROVED** [review]

**Pull request merged**